core/frontend/src/main/scala/io/udash/bindings/TextArea.scala

```diff
@@ -40,4 +42,12 @@ object TextArea {
 
     textarea(bind, xs)
   }
+
+  /**
+    * @param property Property to bind.
+    * @param xs Additional Modifiers, don't use modifiers on value, onchange and onkeyup attributes.
+    * @return HTML textarea with bound Property, applied modifiers and nested options.
+    */
+  def apply(property: Property[String], xs: Modifier*): JsDom.TypedTag[html.TextArea] =
+    apply(property, 20 millis, xs:_*)
```

Comment:
I don't like the debounce enabled by default nor would I expect it after reading this method's doc. How about making this more explicit (maybe `TextArea.debounced` instead of `.apply`)?